hw/ip/sram_ctrl/data/sram_ctrl.hjson

```diff
@@ -201,15 +201,16 @@
         hwaccess: "hro",
         regwen:   "EXEC_REGWEN"
         fields: [
-          { bits: "2:0",
+          { bits: "3:0",
             name: "EN",
+            mubi: true,
             desc: '''
-                  Write 0x5 to this field to enable execution from SRAM.
+                  Write kMubi4True to this field to enable execution from SRAM.
```

Comment:
probably better for this to stay 0x5 in the documents?

Reply:
yeah @msfschaffner and i were discussing this, and we want to switch to a logical representation because if the value changes underneath, it will be really annoying to fix all of this.

I'm in the process of creating the software hearders, see #8663

Reply:
ah, that's a fair point.  Best to use the logic representation for now then.